components/web/aria2/aria2.p5m

```diff
@@ -30,13 +31,12 @@ set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
 
 license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 
-file path=usr/bin/$(MACH64)/aria2c
+file path=etc/bash_completion.d/aria2c
```

Comment:
should we move it to /usr/share/bash-completion/completions/ ?

Reply:
We do now.